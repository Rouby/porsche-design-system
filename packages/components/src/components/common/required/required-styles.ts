import type { Styles } from 'jss';

export const getFunctionalComponentRequiredStyles = (): Styles<'required'> => {
  return {
    required: {
      userSelect: 'none',
    },
  };
};
