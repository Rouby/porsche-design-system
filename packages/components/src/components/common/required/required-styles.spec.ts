import { getFunctionalComponentRequiredStyles } from './required-styles';

describe('getFunctionalComponentRequiredStyles()', () => {
  it('should return correct styles', () => {
    expect(getFunctionalComponentRequiredStyles()).toMatchSnapshot();
  });
});
