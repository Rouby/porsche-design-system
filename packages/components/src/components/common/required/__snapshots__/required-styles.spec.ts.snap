// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getFunctionalComponentRequiredStyles() should return correct styles 1`] = `
{
  "required": {
    "userSelect": "none",
  },
}
`;
